app/assets/stylesheets/search.scss

```diff
@@ -105,6 +105,15 @@
     text-align: center;
   }
 }
+.passenger {
+  border-bottom: 1px solid $light-gray;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 0.85em;
+  vertical-align: center;
+  p{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space